script/deploy/healthCheck.ts

```diff
@@ -506,7 +506,15 @@ const main = defineCommand({
         const safeAddress = networkConfig.safeAddress
         const safeApiUrl = networkConfig.safeApiUrl
         const configUrl = `${safeApiUrl}/v1/safes/${safeAddress}`
-        const res = await fetch(configUrl)
+        let res
+        try {
+          res = await fetch(configUrl)
+        } catch (error) {
+          logError(
+            `Could not fetch SAFE config from configURL ${configUrl}: ${error}`
+          )
+          process.exit(1)
```

Comment:
If `await fetch(configUrl)` fails here the script will log error but never reaches finish(), so we lose the summary of how many errors were found. It also skips any future checks we might add below. As Max mentioned, the lines after the fetch (like safeConfig = await res.json()) will fail if res is undefined.

IMO a good fix is to wrap both the `fetch` and the `res.json()` call in the same try catch and only run the SAFE checks if `safeConfig` is not `null`. That way if the request fails we can log the error but still continue to the rest of our checks

Reply:
this is outdated now as it has been replaced with new code